src/styl/_helpers/_block.styl

```diff
@@ -119,6 +124,18 @@
 .d-iFlex
     display inline-flex
 
+.flx0
+    flex 0
+
+.flx1
+    flex 1
+
+.flx2
+    flex 2
+
+.flx3
+    flex 3
```

Comment:
You can create a function here, checkout how the helper `_spaces` has been done

Reply:
you're right